views/components/form-holder.pug

```diff
@@ -1,9 +1,9 @@
-div(class='content-container intervalForm')
+div(class='content-container intervalForm', id="advSetForm", style="display: none")
```

Comment:
It's not good to have two IDs that are the same words (advanced settings form) but spelled differently. Can you find a way to not use one of the IDs, or perhaps choose a different word (eg advanced settings container)?

Reply:
It doesn't look like id="advSetForm" is being used by anything